AndroidManifest.xml

```diff
@@ -61,6 +61,8 @@
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
     <uses-permission android:name="android.permission.RECEIVE_BOOT_COMPLETED" />
     <uses-permission android:name="android.permission.WAKE_LOCK" />
+    <uses-permission android:name="android.permission.READ_PHONE_STATE" />
```

Comment:
@davigonz I added them when I working on the Issue #2027 (when I was working on master rather than creating a new branch) but I think I forgot to remove it during rebase. I will remove them now.